saleor/graphql/app/schema.py

```diff
@@ -55,9 +58,11 @@ def resolve_apps_installations(self, info, **kwargs):
     def resolve_apps(self, info, **kwargs):
         return resolve_apps(info, **kwargs)
 
-    @permission_required(AppPermission.MANAGE_APPS)
-    def resolve_app(self, info, id):
-        return graphene.Node.get_node_from_global_id(info, id, App)
+    def resolve_app(self, info, id=None):
+        app = info.context.app
+        if not id:
+            return app if app else None
```

Comment:
I'm not sure that we should return None if a user without `MANAGE_APPS` requests for an app without ID.
```suggestion
        if not id and app:
            return app
```